### ISSUE TITLE
Command to give backpack

### ISSUE THREAD

**Issue by SpiderWicket** [GeorgH93/Minepacks#136]
# Feature request

**Feature description**
add a command like /backpack give UserName

**How the feature is useful**
Would be amazing, to be able to use the command to issue backpacks in specific claims, with beauty quests, also even using goldcrates to win the backpack :)

**Comment by Babzy955** [comment]
Yes this would be helpful since i just added this plugin and players are spawning into my server without it
